Source/SelfService/Web/logging/logFilter/filterSelect.tsx

```diff
@@ -2,8 +2,35 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import React from 'react';
+
+import { themeDark } from '../../theme/theme';
 import { Select, SelectProps } from '@mui/material';
 
+const styles = {
+    'padding': '5px 14px',
+    'borderRadius': '4px',
+    'transition': 'all .3s',
+    '&:before, &:hover:not(.Mui-disabled):before, &:after': {
+        border: 'none',
+    },
+    ':hover': {
+        'background': 'rgba(140, 154, 248, 0.08)',
+        '& svg': {
+            color: themeDark.palette.primary.main
+        }
+    },
+    'typography': 'button',
+    '& svg': {
+        right: '5px'
+    },
+    '.MuiSelect-select:focus': {
+        backgroundColor: 'transparent'
+    },
+    '.MuiSelect-iconOpen': {
+        color: themeDark.palette.primary.main
+    },
+};
```

Comment:
I wish I knew what the type of those should be, so that you could get intellisense why accessing the `theme` component in the callbacks. If you figure it out, please do give it a type, if not don't worry, we can revisit it later. My first guess is `SxProps<Theme>` - but this didn't compile.